src/ansys/mechanical/core/embedding/app.py

```diff
@@ -245,9 +241,20 @@ def plot(self) -> None:
             warnings.warn("Plotting is only supported with version 2024R2 and later!")
             return
 
-        from ansys.mechanical.core.embedding.viz.pyvista_plotter import plot_model
+        # TODO Check if anything loaded inside app or else show warning and return
```

Comment:
Is it possible to fix this TODO in this PR?

Reply:
Will fix this later.